bgpd/bgp_main.c

```diff
@@ -449,9 +449,10 @@ int main(int argc, char **argv)
 	if (bgp_port == 0)
 		bgp_option_set(BGP_OPT_NO_LISTEN);
 	bm->address = bgp_address;
-	if (no_fib_flag)
+	if (no_fib_flag) {
 		bgp_option_set(BGP_OPT_NO_FIB);
-
+		bgp_option_set(BGP_OPT_EMULATE_VRF);
```

Comment:
Why is a new flag needed?

Reply:
Because VRF != FIB.
Actually, this is a facility I thought about that.

Woud you agree if I rename BGP_OPT_NO_FIB with BGP_OPT_NO_FIB_EMULATE_VRF ?

Reply:
the no_zebra flags makes sense / seems right to me

Reply:
ok for --no_zebra. if you both agree. I will do the changes.

Reply:
Go for it, I think `--no_zebra` will do the trick.